tests/ovs_test.go

```diff
@@ -27,8 +27,8 @@ var _ = Describe("ovs-cni", func() {
 	Describe("pod availability tests", func() {
 		Context("pod availability tests", func() {
 			It("assert pods exists", func() {
-				pods, _ := clientset.CoreV1().Pods("").List(context.TODO(), v1.ListOptions{})
-				Expect(len(pods.Items)).Should(BeNumerically(">", 0))
+				pods, _ := clientset.CoreV1().Pods("").List(context.TODO(), metav1.ListOptions{ LabelSelector: "app=ovs-cni"})
+				Expect(len(pods.Items)).Should(BeNumerically(">", 0), "should have at least 1 ovs-cni pod deployed")
```

Comment:
nit: maybe s/`Should`/`To` ? 
`Expect(len(pods.Items)).To(BeNumerically(">", 0), "should have at least 1 ovs-cni pod deployed")`

Reply:
Thank you. I'm not sure switching Should-->To will make it more readable..
If for example I induce the Expect to fail, it will look like this:
```
• Failure [0.007 seconds]
ovs-cni
/home/ralavi/go/src/github.com/kubevirt/ovs-cni/tests/ovs_test.go:32
  pod availability tests
  /home/ralavi/go/src/github.com/kubevirt/ovs-cni/tests/ovs_test.go:33
    pod availability tests
    /home/ralavi/go/src/github.com/kubevirt/ovs-cni/tests/ovs_test.go:34
      assert pods exists [It]
      /home/ralavi/go/src/github.com/kubevirt/ovs-cni/tests/ovs_test.go:35

      should have at least 1 ovs-cni pod deployed
      Expected
          <int>: 1
      to be <
          <int>: 0

      /home/ralavi/go/src/github.com/kubevirt/ovs-cni/tests/ovs_test.go:37
```

if it would read as `To have at least 1 ovs-cni pod deployed` - I think it'll be less understood. 
But maybe I can try to make the Context more readable.

Reply:
Done